docs/ota-client-guide/modules/ROOT/pages/add-ota-functonality-existing-yocto-project.adoc

```diff
@@ -13,11 +13,16 @@ endif::[]
 :page-order: 6
 :icons: font
 
-If you already have a Yocto-based project that you want to update using {product-name}, you just need to do four things to get started:
+If you already have a Yocto-based project that you want to update using {product-name}, you can perform the four things outlined below. These are the main changes needed to start a functional integration to an existing Yocto project.
```

Comment:
Having the last sentence after "the four things outlined below" might be a bit unclear.

What would you think of a simplified version, like:

"If you already have a Yocto-based project, you can start your functional integration with {product-name} by following these four steps:"

Reply:
@lbonn good idea. I will replace that line with your suggestion.